package.json

```diff
@@ -11,49 +11,49 @@
   "dependencies": {
     "@babel/core": "^7.9.0",
     "@babel/register": "^7.9.0",
-    "@fortawesome/fontawesome-svg-core": "^1.2.26",
-    "@fortawesome/free-brands-svg-icons": "^5.12.0",
-    "@fortawesome/free-regular-svg-icons": "^5.12.0",
-    "@fortawesome/free-solid-svg-icons": "^5.12.0",
-    "@fortawesome/react-fontawesome": "^0.1.8",
-    "algoliasearch": "^4.0.2",
+    "@fortawesome/fontawesome-svg-core": "^1.2.28",
+    "@fortawesome/free-brands-svg-icons": "^5.13.0",
+    "@fortawesome/free-regular-svg-icons": "^5.13.0",
+    "@fortawesome/free-solid-svg-icons": "^5.13.0",
+    "@fortawesome/react-fontawesome": "^0.1.9",
+    "algoliasearch": "^4.1.0",
     "classnames": "^2.2.6",
     "dotenv": "^8.2.0",
-    "gatsby": "^2.20.8",
-    "gatsby-image": "^2.2.34",
+    "gatsby": "^2.20.9",
+    "gatsby-image": "^2.3.1",
     "gatsby-plugin-algolia": "^0.5.0",
     "gatsby-plugin-manifest": "^2.3.3",
-    "gatsby-plugin-offline": "^3.0.27",
-    "gatsby-plugin-postcss": "^2.1.18",
+    "gatsby-plugin-offline": "^3.1.2",
+    "gatsby-plugin-postcss": "^2.2.1",
     "gatsby-plugin-prefetch-google-fonts": "^1.4.3",
-    "gatsby-plugin-react-helmet": "^3.1.16",
+    "gatsby-plugin-react-helmet": "^3.2.1",
     "gatsby-plugin-sharp": "^2.5.3",
-    "gatsby-source-airtable": "^2.0.12",
-    "gatsby-source-filesystem": "^2.1.40",
+    "gatsby-source-airtable": "^2.1.0",
+    "gatsby-source-filesystem": "^2.2.2",
     "gatsby-transformer-sharp": "^2.4.3",
     "lodash": "^4.17.15",
     "prop-types": "^15.7.2",
-    "react": "^16.12.0",
-    "react-dom": "^16.12.0",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
     "react-helmet": "^5.2.1",
-    "react-instantsearch-dom": "^6.3.0",
+    "react-instantsearch-dom": "^6.4.0",
     "styled-components": "^5.0.1",
-    "styled-icons": "^9.3.0"
+    "styled-icons": "^9.5.0"
   },
   "devDependencies": {
     "@babel/plugin-proposal-class-properties": "^7.8.3",
-    "@babel/plugin-proposal-optional-chaining": "^7.8.3",
-    "@babel/plugin-transform-react-jsx": "^7.8.3",
-    "@babel/preset-react": "^7.8.3",
+    "@babel/plugin-proposal-optional-chaining": "^7.9.0",
+    "@babel/plugin-transform-react-jsx": "^7.9.4",
+    "@babel/preset-react": "^7.9.4",
     "@fullhuman/postcss-purgecss": "^1.3.0",
-    "babel-preset-gatsby": "^0.2.29",
+    "babel-preset-gatsby": "^0.2.36",
     "eslint": "^6.8.0",
-    "eslint-config-airbnb": "^18.0.1",
-    "eslint-config-prettier": "^6.10.0",
-    "eslint-plugin-import": "^2.20.1",
+    "eslint-config-airbnb": "^18.1.0",
+    "eslint-config-prettier": "^6.10.1",
+    "eslint-plugin-import": "^2.20.2",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-prettier": "^3.1.2",
-    "eslint-plugin-react": "^7.18.3",
+    "eslint-plugin-react": "^7.19.0",
```

Comment:
could you move these package changes into their own PR?

Reply:
done, made a reset to both package and package-lock